packages/at_demo_data/lib/src/at_demo_credentials.dart

```diff
@@ -627,193 +25,371 @@ List<String> allAtsigns = <String>[
   '@purnima🛠',
   '@sameeraja🛠',
   '@sitaram🛠',
+  // Key cycle 2
+  '@relay1',
+  '@relay2',
+  '@device1',
+  '@cloudvm1',
+  '@gateway1',
+  '@gateway2',
+  '@policy1',
+  '@policy2',
+  '@colin',
+  '@barbara',
+  '@chris',
+  '@gary',
+  '@xavier',
+  '@jeremy',
+  '@curtly',
+  '@gareth',
+  '@don',
+  '@denise',
 ];
 
 /// List of atsigns to test APKAM feature
-List<String> apkamAtsigns = <String>['@srie', '@sachin'];
+List<String> apkamAtsigns = <String>[
+  // Key cycle 1
+  '@srie',
+  '@sachin',
+  // Key cycle 2
+  '@device2',
+  '@cloudvm2',
+];
 
 /// A Map of cram (Challenge Response Authentication Mechanism) keys.
 /// For more information on CRAM, see: [here](https://atsign.dev/docs/functional_architecture/verbs/#cram)
 Map<String, String> cramKeyMap = <String, String>{
   'anonymous': '',
-  '@alice🛠': AliceKeys._aliceCRAMKey,
-  '@ashish🛠': AshishKeys._ashishCRAMKey,
-  '@barbara🛠': BarbaraKeys._barbaraCRAMKey,
-  '@bob🛠': BobKeys._bobCRAMKey,
-  '@colin🛠': ColinKeys._colinCRAMKey,
-  '@egbiometric🛠': EgBiometricKeys._egBiometricCRAMKey,
-  '@egcovidlab🛠': EgCovidLabKeys._egCovidLabCRAMKey,
-  '@egcreditbureau🛠': EgCreditBureauKeys._egCreditBureauCRAMKey,
-  '@eggovagency🛠': EgGovAgencyKeys._egGovAgencyCRAMKey,
-  '@emoji🦄🛠': EmojiKeys._emojiCRAMKey,
-  '@eve🛠': EveKeys._eveCRAMKey,
-  '@jagan🛠': JaganKeys._jaganCRAMKey,
-  '@kevin🛠': KevinKeys._kevinCRAMKey,
-  '@murali🛠': MuraliKeys._muraliCRAMKey,
-  '@naresh🛠': NareshKeys._nareshCRAMKey,
-  '@purnima🛠': PurnimaKeys._purnimaCRAMKey,
-  '@sameeraja🛠': SameerajaKeys._sameerajaCRAMKey,
-  '@sitaram🛠': SitaramKeys._sitaramCRAMKey,
-  '@srie': SrieKeys._srieCRAMKey,
-  '@sachin': SachinKeys._sachinCRAMKey
```

Comment:
Renamed all of these, removing the name from the variable. Example:
`SachinKeys._sachinCRAMKey` > `SachinKeys._cramKey`. 

It's a private variable, so we know (and I've updated) all the places affected (only files linked via part/part of directive) and thus it is also a non-breaking change.

I only did this rename because it made it easier to duplicate and rename the classes